From 872bb6db52a848ad9093e50505f573828246fa48 Mon Sep 17 00:00:00 2001
From: Ulan Aitbay <69082498+enganese@users.noreply.github.com>
Date: Sun, 28 May 2023 14:20:29 +0500
Subject: [PATCH] Update __init__.py (refactored import)

### **Before**:
`from typings import AiColorsResponse`

### **After**:
`from typings import AiColorsResponse`

### **Comment**:
`Now, it perfectly works!`
---
 gpt4free/aicolors/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gpt4free/aicolors/__init__.py b/gpt4free/aicolors/__init__.py
index f3afa265..a69276b8 100644
--- a/gpt4free/aicolors/__init__.py
+++ b/gpt4free/aicolors/__init__.py
@@ -1,7 +1,7 @@
 import fake_useragent
 import requests
 import json
-from typings import AiColorsResponse
+from .typings import AiColorsResponse
 
 
 class Completion:
